docs/tutorials/local_iteration_series/testing_with_artificial_conditions.md

```diff
@@ -0,0 +1,173 @@
+
+:::important
+
+With Simulator Tools we can't specify if inbound packets and outbound packets would experience different conditions - artificial latecny, jitter and packet loss are bi-directional.
+
+:::
+
+:::important
+
+Simulator Tools window settings will not be committed to version control and need to be set up manually on different editor instnaces.
+
+:::
+
+### Debug Builds
+
+Debug builds do allow for the possibility of applying artificial network conditions to the UTP driver, but the Simulator Tools window itself only sets these values in the Editor.
+
+To set the latency, jitter and packet-loss percentage values for develop builds we need the following code to execute before `NetworkManager` attempts to connect (changing the values of the paramters as desired):
+
+```
+#if DEVELOPMENT_BUILD && !UNITY_EDITOR
+        UnityTransport.ClientPacketDelayMs = 120;
+        UnityTransport.ClientPacketJitterMs = 5;
+        UnityTransport.ClientPacketDropRate = 3;
+#endif
+```
+
+## System-wide network conditioners
+
+These tools are useful when we want to test builds as opposed to running multiple editor instances. It is also an option that works for **release** builds. 
+
+:::important
+
+The solutions described below share some common features:
+ - They do not support latency variability over time, so in effect we can't imitate artificial jitter with them.
+ - They are system-wide, thus all the lcoal instances of our game would run under the same network conditions.
+ - Both allow to control the settings for sending and receiving separately.
+
+:::
+
+There are some inherent limitations that come with the system-wide level of application of these tools - they are not ideal for local testing of more than two peers because we can't have peers with different network conditions between them (if that is important for the test ofcourse).
+
+### Clumsy on Windows
+
+Follow the installation instructions on the official [Clumsy Webpage](https://jagt.github.io/clumsy/).
+
+To test the builds with Clumsy:
+ - Run Clumsy
+ - Run instance 1
+ - Run instance 2
+ - At any point you can adjust Clumsy settings and observe the changes in gameplay fidelity
+
+#### Settings quickstart
+ - Lag - that's our primary lever to control artifical latency
+ - Drop - that's our packet loss
+ - Throttle, Out of Order, Duplicate, Tamper - these will manifest as additional latency but should be automatically handled for us by Netcode. 
+   - Since Clumsy doesn't have Jitter functionality - we can emulate jitter (inconsistnent latency over time) by playing with these parameters. Their cumulative effect would produce artifical latency fluctuations.
+
+For further reading please refer to the Details section of the [Clumsy Webpage](https://jagt.github.io/clumsy/) - the settings explanation there goes more into the actual mechanics of each individual setting.
+
+
+### Network Link Conditioner on OSX
```

Comment:
nothing on dnctl?

Reply:
Will add a section on DummyNet for sure.